editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -63,6 +63,21 @@ void EditorAssetLibraryItem::configure(const String &p_title, int p_asset_id, co
 	price->set_text(p_cost);
 }
 
+void EditorAssetLibraryItem::clamp_width(int p_max_width) {
+	int text_pixel_width = title->get_theme_default_font().ptr()->get_string_size(title->get_text()).x * EDSCALE;
```

Comment:
Why theme default font? You should use the font that the link button actually uses.

In this case that would be `title->get_theme_font(SNAME("font"))`. But we should avoid string access. So I suggest you add a new public method to `LinkButton`, say, `LinkButton::get_button_font()` which returns its `theme_cache.font`, and use it here.

Reply:
Great catch! I couldn't figure out how to properly access the theme font, and I completely spaced on the fact that the default font won't always be the font used in the UI. :facepalm:

I added the getter to the `LinkButton` in the update I just pushed. I was initially going to add a setter, but I wasn't sure exactly how the `theme_cache` integrated with the rest of the theming system, so I wasn't sure if setting the font on the cache was appropriate, so I did not implement a setter for it. Please let me know if you want me to add a setter as well. Thanks!

Reply:
You shouldn't add a setter. You should only read from the cache. It is set automatically when appropriate by the control itself.